packages/buckets-node-sync/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/cjs",
+    "rootDir": "./src"
+  },
+  "references": [
```

Comment:
Can you run `npm run references` to make sure this is up to date. I don't need buckets in here, but I do see hub-threads-client, which this package doesn't reference directly.

Reply:
done